Sources/PathBuilder/LibraryContent.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+import DeveloperToolsSupport
+
+@available(macOS 11.0, *)
```

Comment:
This library is available an all the platform. To be able to compile it we need to add checks on all of them.

```suggestion
@available(macOS 11.0, iOS 14.0, tvOS 14.0, watchOS 7.0, *)
```